src/redux/categories/actionTypes.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [OPTIONAL ] Kindly remove all unused code this will make your code mush noise, It is always a good practice to fix all linters errors instead of disabled it 😉 💯